File: src/main/java/com/gmail/borlandlp/minigamesdtools/gui/hotbar/api/HotbarAPI.java
package com.gmail.borlandlp.minigamesdtools.gui.hotbar.api;

import com.gmail.borlandlp.minigamesdtools.gui.hotbar.Hotbar;
import org.bukkit.entity.Player;

import java.util.Map;

public interface HotbarAPI {
    void bindHotbar(Hotbar hotbar, Player player);
    void unbindHotbar(Player player);
    boolean isBindedPlayer(Player player);
    Hotbar getHotbar(Player player);
    Map<Player, Hotbar> getAll();
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/gui/hotbar/api/HotbarAPI.kt
package com.gmail.borlandlp.minigamesdtools.gui.hotbar.api

import com.gmail.borlandlp.minigamesdtools.gui.hotbar.Hotbar
import org.bukkit.entity.Player

interface HotbarAPI {
    fun bindHotbar(hotbar: Hotbar, player: Player)
    fun unbindHotbar(player: Player)
    fun isBindedPlayer(player: Player): Boolean
    fun getHotbar(player: Player): Hotbar
    val all: Map<Player, Hotbar>
}